src/libraries/System.Text.Json/src/System/Text/Json/Schema/JsonSchema.cs

```diff
@@ -26,8 +26,8 @@ internal sealed class JsonSchema
         internal const string MinLengthPropertyName = "minLength";
         internal const string MaxLengthPropertyName = "maxLength";
 
-        public static JsonSchema False { get; } = new(false);
-        public static JsonSchema True { get; } = new(true);
+        public static JsonSchema False => new(false);
+        public static JsonSchema True => new(true);
```

Comment:
If these aren't singletons, do we even need/want the properties anymore?

Reply:
We probably don't, but changing the signature makes for a larger diff.

Reply:
Updated.